halpybot/packages/announcer/announcer.py

```diff
@@ -186,6 +186,15 @@ async def announce(self, announcement: str, args: Dict, client: HalpyBOT):
         except Exception as announcement_exception:
             logger.exception("An announcement exception occurred!")
             raise AnnouncementError(Exception) from announcement_exception
+        if announcement == "PPWK":
+            cmdr: str = args.get("CMDR")
+            if not cmdr:
+                return
+            try:
+                case: Case = client.board.return_rescue(cmdr.casefold())
+            except KeyError:
+                return  # Case Must Not Have Been From Board.
```

Comment:
possible alternative, if board implements `collections.abc.Mapping`
```py
case = client.board.get(cmdr.casefold())
if case:
    await client.board.del_case(case)
```

Reply:
Made own issue, out of scope for here: #350